src/components/KMultiselect/KMultiselect.vue

```diff
@@ -254,8 +258,8 @@
         </KBadge>
         <!-- Always render this badge even if it's hidden to ensure there will be enough space to show it -->
         <KBadge
+          aria-hidden="true"
```

Comment:
question: if this is aria-hidden, why is it a KBadge?

Reply:
This is staging area - we need it there but hidden